Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -181,39 +214,13 @@ private let aStyles =
 private let baseMarginStyles =
   (h1 | h2 | h3 | h4 | h5 | h6 | p | ul | ol) % margin(topBottom: .rem(0.5), leftRight: 0)
 
-private let typeStyles =
-  (Class.pf.type.largeTitle
-    | Class.pf.type.title1
-    | Class.pf.type.title2
-    | Class.pf.type.title3
-    | Class.pf.type.headline
-    | Class.pf.type.subhead) % color(Color.black)
-
-    <> (Class.pf.type.largeTitle
-      | Class.pf.type.title1
-      | Class.pf.type.title2
-      | Class.pf.type.title3
-      | Class.pf.type.subhead) % fontWeight(.w700)
-
-    <> Class.pf.type.largeTitle % (fontSize(.rem(3.998)) <> lineHeight(1.15))
-    <> Class.pf.type.title1 % (fontSize(.rem(2.827)) <> lineHeight(1.25))
-    <> Class.pf.type.title2 % (fontSize(.rem(1.999)) <> lineHeight(1.25))
-    <> Class.pf.type.title3 % (fontSize(.rem(1.414)) <> lineHeight(1.45))
+private let _headline = CssSelector.class("pf-headline")
 
+private let typeStyles =
+  _headline % fontWeight(.w600)
```

Comment:
we dont have a name for weight 600 in design systems. we should prob have `w-100`, `w-200`, etc... classes